src/main/java/Simon.java

```diff
@@ -1,12 +1,21 @@
+import java.util.Scanner;
 public class Simon {
     public static void main(String[] args) {
-        String output = "____________________________________________________________\n" +
+        String inData = "";
+        Scanner scan = new Scanner( System.in );
```

Comment:
Perhaps could be improved by naming it inputScanner since scan could be interpreted as a verb.